### ISSUE TITLE
Incorrect variable name in onerror() in createEventSource()

### ISSUE THREAD

**Issue by dhalbert** [byjg/jquery-sse#4]
Typo? Should `event` be `e` here?

https://github.com/byjg/jquery-sse/blob/master/jquery.sse.js#L89
```
    // Private Method for Handle EventSource object
    function createEventSource(me) {
        me.type = 'event';
        me.instance = new EventSource(me._url);
        me.instance.successCount = 0;

        me.instance.onmessage = me._settings.onMessage;
        me.instance.onopen = function (e) {
            if (me.instance.successCount++ === 0) {
                me._settings.onOpen(e);
            }
        };
        me.instance.onerror = function (e) {
            if (event.target.readyState === EventSource.CLOSED) {   // <--- event should be e?
                me._settings.onError(event);                                        // <--- event should be e?
            }
        };
```

**Comment by byjg** [comment]
Yeah, there is a typo here. I'll fix it!

**Comment by byjg** [comment]
I published a new version 0.1.4 with this fix :)

Thank you @dhalbert !
